lib/uplinkc/project.go

```diff
@@ -35,16 +32,8 @@ func open_project(uplinkHandle C.UplinkRef, satelliteAddr *C.char, apikeyHandle
 
 	scope := uplink.scope.child()
 
-	var opts *libuplink.ProjectOptions
-	if unsafe.Pointer(cProjectOpts) != nil {
-		opts = &libuplink.ProjectOptions{}
-		opts.Volatile.EncryptionKey = new(storj.Key)
-		for i := range cProjectOpts.key {
-			opts.Volatile.EncryptionKey[i] = byte(cProjectOpts.key[i])
-		}
-	}
-
-	project, err := uplink.OpenProject(scope.ctx, C.GoString(satelliteAddr), apikey, opts)
+	// TODO: add project options argument
```

Comment:
hmm, this makes me think we should either
1) remove the C.ProjectOptions type also (like we did with the mobile bindings)
or 2) leave the ProjectOptions struct entirely (across the board) and only remove the EncryptionKey field, leaving it a weird, empty struct (which is still good for forwards compatibility? it's an easy place to add things in the future)

Reply:
Oh I forgot I added that TODO... Hmm I think I actually want to remove the C.ProjectOptions type entirely.